tests/test_interface.rb

```diff
@@ -1444,6 +1417,7 @@ def test_ipv4_pim_sparse_mode
     #   interface Ethernet1/1
     #     ip pim sparse-mode
     #
+    i.switchport_mode = :disabled
```

Comment:
Did you test against all platforms? This minitest has a ton of corner cases thanks to all the linecard peculiarities...

In particular I think this one will fail on my N7k when it's using the default module-types (ie, non-F3 cards), e.g.

```
n7k8(config-if)# int e1/1
n7k8(config-if)# no switchport 
ERROR: Ethernet1/1: requested config change not allowed   <---------------<< !!!!

n7k8(config-if)# sh run vdc
vdc n7k8 id 1
  limit-resource module-type m1 m1xl m2xl f2e 
  allocate interface Ethernet1/1-48
```

The reason is that this e1/1 is an L2-only interface so you can't disable switchport-mode.  

This is probably a bigger problem for us to solve than in this PR but for now can you add a rescue/skip to catch this condition?

:+1: for everything else